src/scheduler/pbs_sched.c

```diff
@@ -1273,3 +1193,127 @@ main(int argc, char *argv[])
 	log_close(1);
 	exit(0);
 }
+
+/**
+ *
+ * @brief
+ *		Compares two sched commands and tells whether they are duplicate or not.
+ *
+ * @param[in]	cmd1	-	command1
+ * @param[in]	cmd2	-      command2
+ *
+ *
+ * @return	int
+ * @retval	1	-	if both commands are duplicate
+ * @retval	0	-	if not duplicate
+ *
+ */
+static int
+compare_sched_cmd(const sched_cmd *cmd1, const sched_cmd *cmd2)
+{
+	if (cmd1 && cmd2) {
+		if (cmd1->jid == NULL && cmd2->jid == NULL) {
+			if (cmd1->cmd == cmd2->cmd)
+				return 1;
+		} else {
+			if (cmd1->cmd == cmd2->cmd && !strcmp(cmd1->jid, cmd2->jid))
+				return 1;
+		}
+	}
+	return 0;
+}
+
+/**
+ * @brief Filters scheduler commands queue and discards duplicates
+ *
+* @param[in] sched_cmds	- scheduler command queue
+ *
+ * @return	ds_queue *
+ * @retval	success	- returns the filtered queue
+ * @reval	 fail -	NULL
+ *
+ */
+ds_queue *
+filter_sched_cmds(ds_queue *sched_cmds)
+{
+	/* sched_cmds_filtered is a  resultant queue which consists of sched commands after discarding
```

Comment:
Looks like there is an extra space between 'a' and 'resultant'

Reply:
This function is not needed now.